app/views/tasks/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<%# in app/views/new.html.erb %>
+<h2> New page </h2>
+
+<%= form_with model: @task, class: 'create-task' do |f| %>
+  <p>Please provide the following information to add your new task:</p>
+
+  <%= f.label :name %>
+  <%= f.text_field :name %>
+
+  <%= f.label :description %>
+  <%= f.text_field :description %>
+
+   <%= f.label :completed_at %>
+  <%= f.text_field :completed_at %>
```

Comment:
Consider whether you want to input this in a form or default to something like `nil` when a task is created.